### ISSUE TITLE
Error while yielding variable named `module` 

### ISSUE THREAD

**Issue by rilut** [cherow/cherow#230]
**Code**

```js
c.parseScript(`
class C {
    *gen(id, module) {
        yield module.mount(this)
    }
}`) // errors

c.parseScript(`
class C {
    *gen(id, module) {
        yield module
    }
}`) // errors too

c.parseScript(`
class C {
    *gen(id, modul) {
        yield modul.mount(this)
    }
}`) // okay
```

**Expected behavior:**

yielding a variable named `module` should not error

**Actual behavior:**

SyntaxError: Line 4, column 14: Unexpected token 'module'

**Comment by KFlash** [comment]
Are you using the current Master branch? Or 1.6.8?  V. 2.0 is currently a WIP

**Comment by KFlash** [comment]
This works in v. 1.6.8. See [online demo](https://cherow.github.io/cherow/?code=c.parseScript(%60%0Aclass%20C%20%7B%0A%20%20%20%20*gen(id%2C%20module)%20%7B%0A%20%20%20%20%20%20%20%20yield%20module.mount(this)%0A%20%20%20%20%7D%0A%7D%60)%20%2F%2F%20errors&method=parse&range=true&loc=undefined&next=false&module=false&raw=false&jsx=undefined&directives=undefined&attachComment=undefined&early=false)

**Comment by rilut** [comment]
I'm using 1.6.8 on Node v9.8.0. Yes you're right, it works on the online demo. 

I have no idea why it errors on my node:

```js
> c.version
'1.6.8'
> c.parseScript(`
... class C {
.....     *gen(id, module) {
.......         yield module.mount(this)
.......     }
..... }`)
SyntaxError: Line 4, column 14: Unexpected token 'module'
```

**Comment by KFlash** [comment]
Closing as fixed in the upcoming 2.0
